File: 30-Days-of-Code/Day-3-Intro-to-Conditional-Statements/CSharp/solution.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
class Solution {

    static void Main(String[] args) {
        int N = Convert.ToInt32(Console.ReadLine());
        
        if (N % 2 == 1)
            Console.WriteLine("Weird");
        else if (N >= 6 && N <= 20)
            Console.WriteLine("Weird");
        else
            Console.WriteLine("Not Weird");
    }
}

File: 30-Days-of-Code/Day-3-Intro-to-Conditional-Statements/Scala/solution.scala
object Solution {

    def main(args: Array[String]) {
        val sc = new java.util.Scanner (System.in);
        var N = sc.nextInt();
        
        if ( N%2 == 1 ) {
            println("Weird");
        } else if ( N >= 6 && N <= 20 ) {
            println("Weird");
        } else {
            println("Not Weird");
        }
    }
}
